dags/hacker_news.py

```diff
@@ -12,38 +12,24 @@
 ) 
 from src.parse_data import build_stories_info   # type: ignore
 
-logger = logging.getLogger('hackerNews')
-
-# Logger setup
-logging.config.dictConfig(
-    {
-        'version': 1,
-        'disable_existing_loggers': False,
-        'handlers': {
-            'console': {
-                'level': 'INFO',
-                'class': 'logging.StreamHandler'
-            }
-        },
-        'loggers': {
-            'hackerNews': {
-                'handlers': ['console']
-            },
-            'parseStoriesInfo': {
-                'handlers': ['console']
-            }
-        }
-    }
-)
+
+logger = logging.getLogger('airflow.task')
+logger.setLevel(logging.INFO)
+
+FILENAME = os.environ.get('BUCKET_MINIO_FILENAME', '')
+BUCKET_PATH = os.environ.get('BUCKET_MINIO_PATH', '')
+BUCKET_NAME = os.environ.get('BUCKET_MINIO_NAME', '')
```

Comment:
May be it is worth just for readability to separate settings/constants into separate module and import them from there?